src/SmartComponents/BaselinesPage/EditBaseline/EditBaseline.js

```diff
@@ -11,15 +11,17 @@ import {
     TableTextInput
 } from '@patternfly/react-inline-edit-extension';
 import { Skeleton, SkeletonSize } from '@redhat-cloud-services/frontend-components';
+import { AddCircleOIcon } from '@patternfly/react-icons';
 
 import { baselinesPageActions } from '../redux';
 import { baselinesTableActions } from '../../BaselinesTable/redux';
-import api from '../../../api.js';
```

Comment:
what did this do previously?

Reply:
We were running a patch directly from the api. Now we call an action that calls the api so we can have the data in the payload.